ember-select-light/addon-main.cjs

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+const { addonV1Shim } = require('@embroider/addon-shim');
```

Comment:
This looks good as far as avoiding a major SemVer bump goes, provided no other major code changes were required to accommodate the migration.